azkaban-common/src/main/java/azkaban/project/AzkabanProjectLoader.java

```diff
@@ -49,6 +48,7 @@
 class AzkabanProjectLoader {
```

Comment:
As you refactor, it would be good to add missing class docs.

Reply:
New code should follow the dev guide as much as possible.

Reply:
Will change it, thanks!